hanadb_exporter/main.py

```diff
@@ -24,7 +24,11 @@
 from hanadb_exporter import db_manager
 
 LOGGER = logging.getLogger(__name__)
-CONFIG_FOLDER = '/etc/hanadb_exporter'
+# in new systems /etc/ folder is not used in favor of /usr/etc
+INIT_FILES = [
```

Comment:
I wouldn't name this `INIT_FILES`. I would name it `CONFIG_FILES`. that is what it stores.